src/main/resources/avro/genotypephenotypemethods.avdl

```diff
@@ -0,0 +1,115 @@
+
+@namespace("org.ga4gh")
+/**
+This protocol defines genotype to phenotype relationships
+query system.
+
+The goal here is to allow users to query systems using one
+or more of Genotype, Phenotype, Environment, and Evidence.
+
+A query using one of the above items is to mean, by default,
+that the remainder of the query is as a "wildcard", such
+that all matches to just that query term would come back.
+Combinations of the above are to act like AND rather than OR.
```

Comment:
Since each `FeaturePhenotypeAssociation` can have only one `evidence` specified, and multiple `EvidenceQuery`s are ANDed together, does that mean if I specify two types of evidence in my search I will get no results back (since no `FeaturePhenotypeAssociation` can have both the types of evidence I asked for)?